pax/src/main.rs

```diff
@@ -694,7 +698,20 @@ pub fn bundle(
     let writer = Writer {
         modules: modules
             .into_iter()
-            .map(|(k, ms)| (k, ms.unwrap()))
+            .map(|(k, ms)| {
+                (
+                    if k.as_path().starts_with(entry_point.parent().unwrap()) {
+                        PathBuf::from(
+                            k.as_path()
+                                .strip_prefix(entry_point.parent().unwrap())
+                                .unwrap(),
+                        )
+                    } else {
+                        k
+                    },
+                    ms.unwrap(),
+                )
+            })
```

Comment:
this might be more idiomatic as

```
let parent = entry_point.parent().unwrap();

if let Ok(path) = PathBuf::from(k.as_path().strip_prefix(parent)) {
  return (path, ms.unwrap());
else {
  return (k, ms.unwrap());
}
```

Reply:
or this:

```rust
let parent = entry_point.parent().unwrap();
let ms = ms.unwrap();

match PathBuf::from(k.as_path().strip_prefix(parent)) {
  Ok(path) => (path, ms),
  Err => (k, ms)
}
```